### ISSUE TITLE
Errors out on checksum mismatch for ImageMagick7 (and 6)

### ISSUE THREAD

**Issue by jeffmcneill** [skycocker/chromebrew#2144]
## Description
Provide a short description of your problem here.

Unable to attempt to download source once it is downloaded and a checksum mismatch has happened. This is for ImageMagick7. Same error on ImageMagick6. It has an error after downloading and installing all dependencies. Note that I was able to install on another machine, so this could be an error I simply did not encounter in that installation, upstream error, or something else.

## Expected Behavior

Attempt to download source again will delete the downloaded file(s) and start the download again.

## Actual Behavior

Attempt to download again terminates too quickly to be an attempt to download again. 

~~~
crew install imagemagick7
imagemagick7: Use ImageMagick to create, edit, compose, or convert bitmap images.
http://www.imagemagick.org/script/index.php
version 7.0.7-24
No precompiled binary available for your platform, downloading source...
Checksum mismatch. :/ Try again.
~~~

## Versions and system information
chromebrew: 0.4.3

CHROMEOS_ARC_ANDROID_SDK_VERSION=25
CHROMEOS_ARC_VERSION=4613139
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={564AC308-CBD5-485D-9EF5-EB97DBB5F264}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={564AC308-CBD5-485D-9EF5-EB97DBB5F264}
CHROMEOS_RELEASE_BOARD=bob-signed-mpkeys
CHROMEOS_RELEASE_BRANCH_NUMBER=76
CHROMEOS_RELEASE_BUILDER_PATH=bob-release/R64-10176.76.0
CHROMEOS_RELEASE_BUILD_NUMBER=10176
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=64
CHROMEOS_RELEASE_DESCRIPTION=10176.76.0 (Official Build) stable-channel bob 
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_VERSION=10176.76.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=10176.76.0

## Addtional information
Mention things we might need to know. Like:

What I think needs to be done:
- [ ] step one
- [ ] step two

---

## That's it
Thank you for submitting your issue.
When done, please delete the parts of this template which you don't need or these, which are only for guidance.

**Comment by uberhacker** [comment]
Thank you for reporting @jeffmcneill.  The reason it fails the checksum is because that version is no longer available for download.  The latest versions are:
```
https://www.imagemagick.org/download/releases/ImageMagick-6.9.9-39.tar.xz
https://www.imagemagick.org/download/releases/ImageMagick-7.0.7-27.tar.xz
```
With that said, I wasn't able to generate binaries so some more work is needed to make these packages available.

**Comment by jeffmcneill** [comment]
I'm pretty sure the last time I installed it was also from source. Currently I am stuck after downloading ImageMagick-7.0.7-27.tar.xz, as I can't follow the ./configure step at http://www.imagemagick.org/script/install-source.php because of rights issues. 

chromebrew was very helpful in the past since it handled installation from source when it was pointing to a version that existed. 

Is it just a matter of updateding the .rb to get the standard behavior out of crew? I've submitted a pull request with the updated info, if that is the case.
